### PR TITLE
updated database models and README file

### PR DESCRIPTION
**Pull request by swaraj7**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,62 @@
...
+
+`POST /snippet`
+
+**Input**
+
+```
+{
+    "snippet_name" : "unique name for the snippet",
+    "snippet_privacy" : true, 
+    "snippet_expiry" : 5D,
```
Comment (vishesh92) [inline README.md]: `5D` should be string.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,62 @@
...
+**Input**
+``` 
+{
+    "snippet_id" = 1234
+}
+```
+
+**Output**
+```
+Status : 302 Found
```
Comment (vishesh92) [inline README.md]: Fix status code.

File: README.md [hunk 3]
```diff
@@ -1,23 +1,62 @@
...
+  "snippet_expiry" : "5D",
+}
+```
+
+## To access the snippet from link
+
+`GET /snippet/id`
+
+**Input**
+```
```
Comment (vaidik) [inline README.md]: This input is not required as you will be passing your `id` in the URL scheme.

File: README.md [hunk 4]
```diff
@@ -1,23 +1,52 @@
...
+`POST /snippet`
+
+**Input**
+
+```
+{
+    "snippet_name" : "unique name for the snippet",
+    "snippet_privacy" : true, 
+    "snippet_expiry" : "5D",  
+    "snippet_file" : {
```
Comment (vaidik) [inline README.md]: You can improve this. Is the file name really required in a snippet? You should change this with `snippet_content`.

**Pull request merged**